sycl/test/matrix/matrix-check-types-in-attributes.cpp

```diff
@@ -0,0 +1,52 @@
+// RUN: %clangxx -fsycl -fsycl-device-only -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 -O2 -S -emit-llvm -o - %s | FileCheck %s
```

Comment:
Please fix CODEOWNERS in a separate PR to add "sycl/test/matrix" to matrix reviewers ownership. Once merged in a separate PR, this one shouldn't require SYCL RT approval anymore (I think).

Reply:
Created https://github.com/intel/llvm/pull/11564